mudlet-mapper.xml

```diff
@@ -4882,6 +4882,10 @@ function mmp.startup()
   private_settings[&quot;universe&quot;]      = createOption(false, mmp.setUniverse, {&quot;boolean&quot; }, &quot;Use Universe Tarot?&quot;)
   private_settings[&quot;gare&quot;]      		= createOption(false, mmp.setGare, {&quot;boolean&quot; }, &quot;Use Gare?&quot;)
 
+--Imperian wings
+	private_settings[&quot;shekinah&quot;]     	= createOption(false, mmp.setShekinah, { &quot;boolean&quot; }, &quot;Use Seraphim Wings?&quot;)
+	private_settings[&quot;suriel&quot;]     		= createOption(false, mmp.setSuriel, { &quot;boolean&quot; }, &quot;Use Orphanim Wings?&quot;)
```

Comment:
Could you run the code formatter on this script to get rid of the stranhe indenting here?

Reply:
I'm not seeing the odd indenting with this script open in mudlet - All looks neat?

Reply:
It's a tabs vs. spaces issue here